emscripten-simulated-jerry-api/jerry.c

```diff
@@ -289,12 +289,11 @@ void jerry_release_value (jerry_value_t value) /**< API value */
 jerry_value_t
 jerry_run_all_enqueued_jobs (void)
 {
-  EM_ASM (
+  return (jerry_value_t) EM_ASM_INT (
```

Comment:
Can the unit test that runs `jerry_run_all_enqueued_jobs` be re-enabled again now?

Reply:
no, unless we wrap the promise here?

Reply:
Hmm, now I think we can wrap promise here in module.js instead of in certain project. It makes sense because developers should control the promise queue manully